code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -366,10 +366,10 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 		return ..()
 
 /mob/living/simple_animal/hostile/guardian/death(gibbed)
-	. = ..()
 	if(!QDELETED(summoner))
 		to_chat(summoner, span_bolddanger("Your [name] died somehow!"))
 		summoner.dust()
+	..()
```

Comment:
```suggestion
	return ..()
```

Reply:
Yes and also the (pretty unlikely) off chance that someone makes it return a value in the future